config/environments/development.rb

```diff
@@ -53,4 +53,6 @@
   # Config::Options objects, but we need them as hashes.
   cache_settings = Settings.cache.to_hash
   config.cache_store = cache_settings[:store].to_sym, *cache_settings[:opts]
+
+  GA.tracker = Settings.googleanalytics.tracker
```

Comment:
Elsewhere there are references to a `GoogleAnalytics` module. Can you clarify where the `GA` module is defined?